float32 setF
---
float32 newResult